packages/twenty-front/src/testing/graphqlMocks.ts

```diff
@@ -343,3 +343,37 @@ export const graphqlMocks = {
     }),
   ],
 };
+
+export const userMetadataLoaderMocks = {
```

Comment:
I don't think this is the right place, this looks this might be local overwrite and should only be part of the interested set of stories

Reply:
Alright!